### PR TITLE
Add stream management HTTP api

### PR DESCRIPTION
**Pull request by PawelPeczek-Roboflow**
# Description

Please see documentation that was created!

## Type of change

Please delete options that are not relevant.

-   [x] New feature (non-breaking change which adds functionality)
-   [x] This change requires a documentation update

## How has this change been tested, please provide a testcase or example of how you tested the change?

* suite of unit tests
* end 2 end on CPU machine
* planned: e2e on Jetson and GPU - results will be delivered soon

## Any specific deployment considerations
* https://roboflow.slack.com/archives/C066S87TWH1/p1702301407743869

## Docs

-   [x] Docs updated? What were the changes:

### CODE REVIEW

**Comment by PawelPeczek-Roboflow** [comment]
Verified that it works under Linux/GPU

**Comment by PawelPeczek-Roboflow** [comment]
Confirmed that it works e2e at jetson device

File: .github/workflows/docker.stream_management_api.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+name: Build and Push CPU Container
```
Comment (paulguerrie) [inline .github/workflows/docker.stream_management_api.yml:1]: Update name to deconflict with existing CPU container workflow

**Review by paulguerrie — APPROVED** [review]
Looks good to me! Left some comments, but no show stoppers.

File: .github/workflows/docker.stream_management_api.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (PawelPeczek-Roboflow) [reply .github/workflows/docker.stream_management_api.yml:1]: 👍

File: .github/workflows/docker.stream_manager.cpu.yml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: Build and Push CPU Container
```
[the remark this answers is not in the document]
Comment (PawelPeczek-Roboflow) [reply .github/workflows/docker.stream_manager.cpu.yml:1]: 👍

File: .github/workflows/docker.stream_manager.gpu.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+name: Build and Push GPU Container
```
[the remark this answers is not in the document]
Comment (PawelPeczek-Roboflow) [reply .github/workflows/docker.stream_manager.gpu.yml:1]: 👍

File: .github/workflows/docker.stream_manager.jetson.5.1.1.yml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+name: Build and Push Jetson 5.X Container
```
[the remark this answers is not in the document]
Comment (PawelPeczek-Roboflow) [reply .github/workflows/docker.stream_manager.jetson.5.1.1.yml:1]: 👍

File: docker/dockerfiles/stream-management-api.compose-cpu.yaml
[the remark this answers is not in the document]
Comment (PawelPeczek-Roboflow) [reply docker/dockerfiles/stream-management-api.compose-cpu.yaml:1]: we could although I just separated based on the common good practices: https://docs.docker.com/config/containers/multi-service_container/

**Review by paulguerrie — APPROVED** [review]

**Pull request merged**